src/utils/iterateFocusable.ts

```diff
@@ -0,0 +1,33 @@
+/**
+ * Returns an iterator over all of the focusable elements within `container`.
```

Comment:
Why not return an array instead of an iterator if we'll most likely need to convert it to an array in use anyways?

Reply:
The DOM can be quite large sometimes - thousands of elements. If you only need the first element, for example, there's no reason to walk the entire DOM and construct a huge array. Using an iterator here allows us to walk one element at a time.

Reply:
Ah makes sense, though this function only walks through the DOM elements inside of the container right?

I think I just got thrown off by the tests, I was thinking we'd be wanting to grab the whole array of focusable items in our focus trap behvavior, but if that we would just be to calling`iterateFocusableElements(container, true).next()` whenever the user presses tab inside of the container, correct?

Reply:
Yep, all correct!